browser/resources/settings/brave_content_page/content_page_index.html

```diff
@@ -32,6 +32,15 @@
     </settings-brave-content-playlist>
   </template>
 
+  <template is="dom-if" if="[[showPage_(pageVisibility_.playlist)]]">
+    <settings-brave-content-playlist
+      id="playlist"
+      slot="view"
+      prefs="{{prefs}}"
+      in-search-mode="[[inSearchMode_]]">
+    </settings-brave-content-playlist>
+  </template>
```

Comment:
This looks to be a duplicate of the template above.

Reply:
Yes, thank you, fixed here https://github.com/brave/brave-core/pull/29767/commits/288506a1d6762b7d6cb1c6f33fe5f481f100b878